setup/docker-compose.tests.yml

```diff
@@ -1,14 +1,14 @@
 version: "3"
 services:
   web-server:
+  #builds from tests/dockerfile
     build: tests
     depends_on:
       - db
     environment:
       - DB_HOST=db
       - WEB_SERVER_HOST=0.0.0.0
-    ports:
-      - "8080:8080"
```

Comment:
It's true that we don't need this, but it would be good to have a comment explaining why.
Ideally this would have been in the commit https://github.com/nataliejschultz/e-mission-server/commit/dbce159cfcb341f1ade5d0d5681f8cd3ac19d1d1 but we can at least put it in the PR now...

Reply:
Let me know if this is true: The port is only necessary if the container is being called from somewhere else (ie when the server calls nominatim's container, it needs the port). Otherwise, the network function of GitHub actions takes care of keeping the containers separate.